robot.py

```diff
@@ -35,20 +38,25 @@ def createObjects(self):
         self.operator_joystick = wpilib.Joystick(1)
 
     def teleopPeriodic(self):
+        '''makes robot direction correspond to joystick direction'''
         self.drivetrain.turn_at(
             self.drive_joystick.getRawAxis(0), squaredInputs=True)
         self.drivetrain.forward_at(self.drive_joystick.getRawAxis(1))
 
         if self.drive_joystick.getRawButton(4):
             self.bunny_grabber.set_forward()
+            '''button 4 on joystick moves the bunny grabber forward'''
```

Comment:
These comments seem to be repeating the code that they are next to. Do they need to be there?